x-pack/plugins/search_playground/public/components/summarization_panel/summarization_model.tsx

```diff
@@ -94,6 +96,10 @@ export const SummarizationModel: React.FC<SummarizationModelProps> = ({
     [models]
   );
 
+  useEffect(() => {
+    usageTracker.click(`${AnalyticsEvents.modelSelected}_${selectedModel}`);
```

Comment:
model selected event issue. Also worth not putting the name in the event, can we pass it in as an attribute instead?
```
                {
                  "appName": "search_playground",
                  "eventName": "model_selected_[object Object]",
                  "lastUpdatedAt": "2024-05-01T09:37:43.854Z",
                  "fromTimestamp": "2024-05-01T00:00:00Z",
                  "counterType": "click",
                  "total": 1
                },
```